src/applications/bmqstoragetool/m_bmqstoragetool_compositesequencenumber.h

```diff
@@ -139,19 +138,20 @@ bool operator<=(const CompositeSequenceNumber& lhs,
 
 // ACCESSORS
 
-inline bool CompositeSequenceNumber::isSet() const
+inline unsigned int CompositeSequenceNumber::leaseId() const
 {
-    return d_isSet;
+    return d_compositeSequenceNumber.first;
 }
 
-inline unsigned int CompositeSequenceNumber::leaseId() const
+inline bsls::Types::Uint64 CompositeSequenceNumber::sequenceNumber() const
 {
-    return d_leaseId;
+    return d_compositeSequenceNumber.second;
 }
 
-inline bsls::Types::Uint64 CompositeSequenceNumber::sequenceNumber() const
+inline bsl::pair<unsigned int, bsls::Types::Uint64>
```

Comment:
Can return const ref to the field and avoid making a copy

Reply:
Fixed as proposed.